docs/dm/attestation-results/attestation-results.cddl

```diff
@@ -0,0 +1,44 @@
+start = attestation-results
+
+$ar-status /= ar.status-failure
+$ar-status /= ar.status-success
+$ar-status /= ar.status-unknown
+
+attestation-results = {
+  ar.status => $ar-status
+  ? ar.trust-vector => ar-trust-vector
+  ? ar.raw-evidence => ar-bytes
+  ar.timestamp => ar-timestamp
+  ? ar.endorsed-claims => ar-endorsed-claims
+  * $$attestation-results-extension
+}
+
+ar-trust-vector = {
+  ar.hw-authenticity => $ar-status
+  ar.sw-integrity => $ar-status
+  ar.sw-up-to-dateness => $ar-status
+  ar.config-integrity => $ar-status
+  ar.runtime-integrity => $ar-status
+  ar.certification-status => $ar-status
+  * $$ar-trust-vector-extension
+}
+
+ar-endorsed-claims = non-empty<{
+  ? ar.hw-details => ar-hw-details
+  ? ar.sw-details => ar-sw-details
+  ? ar.certification-details => ar-certification-details
```

Comment:
Should there not be an extension point to the ar-endorsed-claims in-line with trust vector extension ?

Reply:
makes sense!

Reply:
Thank You!